cmd/ltctl/comparison.go

```diff
@@ -67,6 +67,13 @@ func writeResults(results []comparison.Result, writer io.Writer) error {
 	var content string
 
 	for i, res := range results {
+		if len(res.LoadTests) < 2 || res.LoadTests[0].Failed || res.LoadTests[1].Failed {
```

Comment:
So if one failed, and other passed, we would not be displaying the one that passed right? Is that intentional? I was thinking we can move this check inside the for loop and just ignore the one that failed.

Reply:
The thing is that if a single test has failed, there are undefined things here, like the `DashboardURL` (which contains the queries from the base and new builds, one of them with an offset to compare both runs in the same graph). Those are generated in `Comparison.getResults`, which contains the same check to skip the generation of those results. We could technically avoid writing the results of these common things and print whatever individual results we have, but what I decided here was to treat a comparison like an indivisible unit: given that we want to compare two things, in my opinion there should be no results if both tests didn't finish successfully.